ZuluContent/Zulu/Engines/Magic/Enums/CurseType.cs

```diff
@@ -0,0 +1,10 @@
+namespace ZuluContent.Zulu.Engines.Magic.Enums
+{
+    public enum CurseLevelType
+    {
+        None = 0,
+        UnRevealed,
```

Comment:
Bit of a nitpick but the namings seem odd to me, usually you don't title case prefixes on words, e.g. `Unrevealed` is a word by itself so you wouldn't capitalize the R.
Also I'd be inclined to just have `Revealed` instead of `RevealedCanUnEquip`. Mostly because I could see people reading the code and getting confused because there's only 1 character difference in the names:
```
        RevealedCantUnEquip,
        RevealedCanUnEquip,
```

Reply:
Yeah true ok I'll change it